inc/classes/CDN/RocketCDN/APIClient.php

```diff
@@ -0,0 +1,219 @@
+<?php
+namespace WP_Rocket\CDN\RocketCDN;
+
+/**
+ * Class to Interact with the RocketCDN API
```

Comment:
Make sure that all descriptions have a full hard stop, i.e. end in a period. Why? To comply with WPCS. Please change this throughout the PR.

Reply:
PHPCS/WPCS analysis doesn't give me any warning/error about class/methods description needing full stop, only for parameters descriptions. Am I missing something?

Reply:
I haven't looked yet at our WPCS configuration. But one of the WP rulesets is really fussy about full stops.

What do you think about making it part of our coding standard?

Reply:
I don't have a strong opinion on it, I'm personally loose with full stops. So it can be added if the majority of the dev team is in favour of it

Reply:
We'll circle back to this and the rest of our codebase once we get agreement on our coding standards and update `phpcs.xml` configuration.